src/template_storage.mligo

```diff
@@ -0,0 +1,10 @@
+// SPDX-FileCopyrightText: 2021 TQ Tezos
+// SPDX-License-Identifier: LicenseRef-MIT-TQ
+
```

Comment:
Much like `template.mligo` wouldn't it be best for this (and each implementation) to have a check that it wasn't imported already?
Like:
```
#if !VARIANT_STORAGE
#define VARIANT_STORAGE
```
or something like that

Reply:
Yes, done.